xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/layouts/livedata-cards.html

```diff
@@ -51,10 +54,10 @@
  */
 define([
   "Vue",
-  "dropdownMenu",
+  BASE_PATH + "topbar.js",
```

Comment:
You're inconsistent with the "topbar" naming. In some places you use camel case and in others you use lowercase. And I've seen one place with "top-bar". I'd settle to **camel case** everywhere.